esmvalcore/cmor/table.py

```diff
@@ -562,9 +585,16 @@ def get_variable(self, table, short_name):
 
         """
         var_info = self.tables.get(table, {}).get(short_name, None)
-        if not var_info and self.default:
-            return self.default.get_variable(table, short_name)
-        return var_info
+        if var_info:
```

Comment:
`if var_info is not None:`

Reply:
it works like this, no need to double check

Reply:
There is no double check. What do you mean?

Reply:
@valeriupredoi ditto?